dockerfiles/debian-jessie-all/Dockerfile

```diff
@@ -2,7 +2,11 @@
 FROM debian:jessie
 
 # install build tools and PostgreSQL development files
-RUN apt-key adv --keyserver ha.pool.sks-keyservers.net --recv-keys B97B0AFCAA1A47F044F244A07FCC7D46ACCC4CF8 \
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends \
```

Comment:
Is there a reason these can't be added to the `apt-get install` step below (which already has a long list of alphabetized package names?)

Reply:
It was there because they are needed for apt-key command.